### PR TITLE
Issue2 view entries

### PR DESCRIPTION
**Pull request by m-Cho1**
code-journal-feature-2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,11 +66,28 @@ <h2>New Entry</h2>
...
-            <input type="submit" value="SAVE" class="style-button">
+            <input type="submit" value="SAVE" class="style-button" id="button-save">
           </div>
         </div>
       </form>
     </div>
+
+    <div data-view="entries" id="view-entries" class="view hidden container display-flex flex-direction">
+      <div class="row align-items">
+        <div class="row-half">
```
Comment (codingcodymiller) [inline index.html:77]: `row-half`? I would think you want `column-half`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
Edited!

File: css/styles.css [hunk 2]
```diff
@@ -95,7 +99,7 @@ img {
   padding-left: 30px;
 }
 
-.row-half {
+.column-half-1 {
```
Comment (codingcodymiller) [inline css/styles.css:102]: Why `-1`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
Edited!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**